src/data_processing/file_paths.py

```diff
@@ -17,13 +17,16 @@ def build_calibrated_file_name(self, instrument, makedir=True):
         return self._get_nc_name(folder, identifier)
 
     def build_standard_output_file_name(self, product='categorize', makedir=True):
-        folder_id = 'processed' if product == 'categorize' else 'products'
-        root = self.config['main']['PATH']['output']
-        folder = os.path.join(root, self.site_name, folder_id, product, self._year)
+        folder = self.build_standard_output_path(product)
         if makedir:
             os.makedirs(folder, exist_ok=True)
         return self._get_nc_name(folder, product)
 
+    def build_standard_output_path(self, product='categorize'):
```

Comment:
Pitäisikö näissäkin funktioissa olla tyyppivinkit?

Reply:
Joo näitä olisi hyvä lisätä sitä mukaa kun ehtii kaikkialle.